include/util/debug_geometry.hpp

```diff
@@ -69,6 +71,8 @@ boost::filesystem::ofstream dg_debug_turns_file;
 bool dg_output_turn_debug = false;
 bool dg_first_turn_debug = true;
 
+std::unordered_map<OSMNodeID, util::FixedPointCoordinate> node_lookup_map;
```

Comment:
@MoKob you're still right in that is undefined behavior, violating the ODR (one definition rule). If multiple TUs include that header and then get linked together :boom:

Needs an `extern` declaration in the header and a single definition in exactly one implementation file.